cypress/integration/login.js

```diff
@@ -1,32 +1,42 @@
+import LoginPage from '../objects/loginPage';
+import TimeLoggingPage from '../objects/timeLoggingPage';
+import PageComponents from '../objects/pageComponents';
+import Common from '../objects/common';
+
+const loginPage = new LoginPage();
+const timePage = new TimeLoggingPage();
+const pageComp = new PageComponents();

```

Comment:
This is minor, but i would not shorten the names of variables and name this `pageComponents`